### ISSUE TITLE
Accessible name and visible label mismatch in menu items

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3587]
**Issue Description**: The first words of the accessible name must match the visible label. Also, the role information does not need to be repeated in the name because it will be announced as part of the element's description, and including "undefined" in the name is a point of confusion. The text in the menu item here should be the same as the text in the aria-label attribute.  The text of the element could also provide the name to make sure they match.

**WCAG Criteria**: [2.5.3 Label in Name - A](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name)

<img width="1402" alt="Screenshot 2024-08-08 at 9 43 39 AM" src="https://github.com/user-attachments/assets/2f3a8b6a-8be9-47e2-a922-05857d62feb7">

**Notes**:

- It is generally sufficient and true in this case that the accessible name matches the visible text of the control/input. "gpt-3.5-turbo-16k-0613" would be the best name in this instance.
- - This issue is repeated throughout the menu in each menu item. 
- One option is to rely on the text of the menu-item to provide the name if this is easier than trying to calculate an aria-label value for the component.

<img width="1519" alt="Screenshot 2024-08-08 at 9 48 44 AM" src="https://github.com/user-attachments/assets/e1248d5f-2bee-4987-b3a1-8088a33a6710">

**Comment by danny-avila** [comment]
Closed by #3630 

![image](https://github.com/user-attachments/assets/e607b674-41e5-456c-ae02-fbcac099f509)
